### PR TITLE
search tokens by address

### PR DESCRIPTION
**Pull request by kev1n-peters**

### CODE REVIEW

**Review by artursapek — DISMISSED** [review]
nice

File: wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenList.tsx [hunk 1]
```diff
@@ -179,31 +179,34 @@ const TokenList = (props: Props) => {
...
 
         const queryLC = query.toLowerCase();
-        return Boolean(
-          token.symbol?.toLowerCase().includes(queryLC) ||
-            getDisplayName(token, props.selectedChainConfig.key)
-              .toLowerCase()
-              .includes(queryLC),
-        );
+
+        const symbolMatch = token.symbol?.toLowerCase().includes(queryLC);
```
Comment (emreboga) [inline wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenList.tsx:197]: Nitpick: If there is a symbolMatch, you can return here without checking other matches. Same as displayNameMatch below. However, if you think having one return at the bottom for all cases makes the code more organized and readable, I'm all for that too 👍

Comment (kev1n-peters) [reply wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenList.tsx:197]: Change to make the code more performant. I don't think readability suffers

**Review by emreboga — APPROVED** [review]
LGTM!

**Pull request merged**